libvast/src/concept/hashable/crc.cpp

```diff
@@ -107,7 +107,7 @@ const uint32_t crc_table[256] = {
 #define DO8(buf) DO4(buf); DO4(buf);
 
 void crc(const void * key, int len, uint32_t seed, void * out) {
-  uint8_t* buf = (uint8_t*)key;
+  auto* buf = (uint8_t*)key;
```

Comment:
I find `auto buf = ...` even more reasonable. Is there a way to configure it that way?

Reply:
> Is there a way to configure clang modernzier that way?

I don't think so. clang-tidy checks also seem not to include C++17 features yet.